src/biome/data/sources/readers.py

```diff
@@ -1,20 +1,49 @@
 import glob
 import logging
 from glob import glob
-from typing import Dict, Optional, Union, List
+from typing import Optional, Union, List
+from urllib.parse import urlparse
 
+import dask
 import dask.dataframe as dd
-import flatdict
 import pandas as pd
 from dask import delayed
-from elasticsearch import Elasticsearch
-from elasticsearch.helpers import scan
+from dask_elk.client import DaskElasticClient
 
-from biome.data.sources.utils import flatten_dataframe
+from biome.data.sources.utils import flatten_dataframe, flatten_dask_dataframe
 
-_logger = logging.getLogger(__name__)
+ID = "id"
+RESOURCE = "resource"
+PATH_COLUM_NAME = "path"
```

Comment:
Small typo: PATH_COLUM_NAME -> PATH_COLUMN_NAME
Also in the `read_csv` method, the `include_path_column` can be a string specifying the column name.
So better to change the `read_csv` call below to:
```python
return dd.read_csv(path, include_path_column=PATH_COLUMN_NAME, **params)
```

Reply:
👍